<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>APM</class>
 <widget class="QMainWindow" name="APM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>465</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>TlwgTypewriter</family>
    <pointsize>14</pointsize>
    <weight>50</weight>
    <italic>true</italic>
    <bold>false</bold>
    <underline>false</underline>
    <strikeout>false</strikeout>
   </font>
  </property>
  <property name="windowTitle">
   <string>无人机地面站</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../build-APM-Desktop_Qt_5_2_0_MinGW_32bit-Debug/debug/icons/airplane.png</normaloff>../build-APM-Desktop_Qt_5_2_0_MinGW_32bit-Debug/debug/icons/airplane.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="font">
    <font>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <widget class="QTabWidget" name="tabwidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>691</width>
      <height>441</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Norasi</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <property name="minimumSize">
      <size>
       <width>697</width>
       <height>360</height>
      </size>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <attribute name="icon">
      <iconset resource="ui.qrc">
       <normaloff>:/icons/dashboard.png</normaloff>:/icons/dashboard.png</iconset>
     </attribute>
     <attribute name="title">
      <string>飞行状态</string>
     </attribute>
     <widget class="QWidget" name="mapWidget" native="true">
      <property name="geometry">
       <rect>
        <x>271</x>
        <y>12</y>
        <width>411</width>
        <height>361</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>411</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>13</y>
        <width>261</width>
        <height>171</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>261</width>
        <height>141</height>
       </size>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>200</y>
        <width>261</width>
        <height>171</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>251</width>
        <height>151</height>
       </size>
      </property>
      <property name="title">
       <string>状态数据</string>
      </property>
     </widget>
     <widget class="QLabel" name="statusLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>370</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="icon">
      <iconset resource="ui.qrc">
       <normaloff>:/icons/icon_cogs.png</normaloff>:/icons/icon_cogs.png</iconset>
     </attribute>
     <attribute name="title">
      <string>参数调节</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="icon">
      <iconset resource="ui.qrc">
       <normaloff>:/icons/paper_airplane.png</normaloff>:/icons/paper_airplane.png</iconset>
     </attribute>
     <attribute name="title">
      <string>航点规划</string>
     </attribute>
     <widget class="QWidget" name="mark_view_widget" native="true">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>461</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>77</y>
        <width>171</width>
        <height>311</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>航点列表</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="locatepushButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>101</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>重新定位</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="icon">
      <iconset resource="ui.qrc">
       <normaloff>:/icons/chart-xy.png</normaloff>:/icons/chart-xy.png</iconset>
     </attribute>
     <attribute name="title">
      <string>数据波形</string>
     </attribute>
     <widget class="QCustomPlot" name="customPlot" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>661</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>330</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>330</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_3">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>330</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Z</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ui.qrc"/>
 </resources>
 <connections/>
</ui>
